pkg/kubelet/container/testing/os.go

```diff
@@ -27,7 +27,7 @@ import (
 // of the real call.
 type FakeOS struct {
 	StatFn     func(string) (os.FileInfo, error)
-	ReadDirFn  func(string) ([]os.FileInfo, error)
+	ReadDirFn  func(string) ([]os.DirEntry, error)
```

Comment:
why not use os.FileInfo?

Reply:
because RealOS.ReadDir and OSInterface.ReadDir  have been modified to return `[]os. DirEntry` .

https://github.com/kubernetes/kubernetes/pull/111221/files#diff-9512da6717e9810d31c29694c8b98e244ed2a69e1995973851253234772057ccR102